dm/pkg/binlog/event/util.go

```diff
@@ -376,6 +379,35 @@ func GetServerCollationByStatusVars(statusVars []byte, idAndCollationMap map[int
 	return idAndCollationMap[int(v)], err
 }
 
+// GetCharsetCodecByStatusVars returns an encoding.Encoding to encode and decode original query if needed.
+func GetCharsetCodecByStatusVars(statusVars []byte) (encoding.Encoding, error) {
```

Comment:
this func called `GetCharsetCodecByStatusVars` but only return chartset code when chartset is gbk , this looks litte strange, maybe change a name?

Reply:
the comment says "returns ... if needed", we can return more codec if user wants

Reply:
ok i see